ConnectWiseManageAPI/Public/System/Documents/New-CWMDocument.ps1

```diff
@@ -18,11 +18,20 @@
             'Rma'
         )]
         [string]$recordType,
+        
```

Comment:
I'd argue that in this instance the line break after this parameter definition makes it easier to see where this parameter finishes and the next one begins.